Rollbar/DTOs/Request.cs

```diff
@@ -14,17 +28,104 @@ public class Request
         /// Initializes a new instance of the <see cref="Request"/> class.
         /// </summary>
         public Request()
-            : this(null)
+            : base(null)
         {
         }
 
+#if (NETSTANDARD || NETCOREAPP)
+
+        public Request(IDictionary<string, object> arbitraryKeyValuePairs, HttpRequest httpRequest = null)
+            : base(arbitraryKeyValuePairs)
+        {
+            if (httpRequest != null)
+            {
+                this.SnapProperties(httpRequest);
+            }
+        }
+
+        private void SnapProperties(HttpRequest httpRequest)
+        {
+            Assumption.AssertNotNull(httpRequest, nameof(httpRequest));
+
+            this.Url = httpRequest.Host.Value + httpRequest.Path;
+            this.QueryString = httpRequest.QueryString.Value;
+            //this.Params = null;
```

Comment:
?

Reply:
uncommenting this one. it is safer to init backing dictionary value with null than forgetting to check on its existence before using it...